zio-http/src/main/scala/zhttp/service/DecodeJResponse.scala

```diff
@@ -11,6 +12,13 @@ trait DecodeJResponse {
   def decodeJResponse(jRes: FullHttpResponse): Client.ClientResponse = {
     val status  = Status.fromHttpResponseStatus(jRes.status())
     val headers = Header.parse(jRes.headers())
-    Client.ClientResponse(status, headers, jRes.content())
+    val content =
+      if (jRes.content().readableBytes() == 0) {
+        Unpooled.EMPTY_BUFFER
+      } else {
+        Unpooled.buffer(jRes.content().readableBytes(), jRes.content().capacity()).writeBytes(jRes.content())
```

Comment:
```suggestion
        Unpooled.copiedBuffer(jRes.content())
```
Will this have the same effect?

Reply:
yes, it will have effect and simplifies our code too